### PR TITLE
Validation file package improvements

### PR DESCRIPTION
**Pull request by josephschorr**

### CODE REVIEW

File: pkg/validationfile/blocks.go [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+		}
+	}
+	return parsed, err
+}
+
+// ParseRelationships parses a newline-separated relationships string into a set
+// of relation tuples.
+func ParseRelationships(relationshipsString string) ([]*v1.Relationship, error) {
+	if relationshipsString == "" {
+		return nil, nil
```
Comment (ecordell) [inline pkg/validationfile/blocks.go:413]: this is kind of a nit but I think I would find it surprising that this doesn't either return an error (no relationships) or an empty slice

File: pkg/validationfile/blocks.go [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	yaml "gopkg.in/yaml.v2"
+	yamlv3 "gopkg.in/yaml.v3"
```
[the remark this answers is not in the document]
Comment (ecordell) [reply pkg/validationfile/blocks.go:12]: That might be worth a quick comment, though I suppose the tests would catch it if someone switched it inadvertently

[the remark this answers is not in the document]
Comment (josephschorr) [reply pkg/validationfile/blocks.go:12]: I added the comment at the usage site, along with a TODO

**Review by ecordell — APPROVED** [review]
LGTM

**Pull request merged**